src/world.rs

```diff
@@ -9,16 +9,18 @@ use crate::world_generation::perlin::PerlinNoise;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use strum::IntoEnumIterator;
+use crate::cubes_to_draw::CubesToDraw;
 
 pub struct World {
     /// The list of the chunks currently being displayed
     chunks: Vec<Chunk>,
+    cubes_to_draw: CubesToDraw,
```

Comment:
This should be an `Option<CubeToDraw>`, because the server will not have this. 

The constructors of this class should create  `cubes_to_draw` with `None`

Then we add a method to instantiate `cubes_to_draw` to something, maybe named `prepare_for_opengl` ? If you have a better name, don't hesitate.